api/src/main/resources/api/service_openapi.yaml

```diff
@@ -20,8 +20,149 @@ paths:
         500:
           description: Service is broken
 
+  # TODO implement "real" export endpoints
+  '/entityFilter/sqlQuery':
```

Comment:
I like POST here because of the detailed body parameterization, and the fact that it's not a standard REST endpoint. Generally I like to suggest following this style guide: https://cloud.google.com/apis/design/custom_methods , though we've only really partially standardized on it in AoU (for example, we don't use `:` in our method names).

If you like that guide, and think it makes sense for Tanagra, then I would suggest something like `/.../generateSqlQuery`, i.e. including a verb. This can also make for a consistent operation ID that looks more like a method name `entitySqlQuery` -->  `generateSqlQuery`

Let me know if there are any other style guides etc that you have from PF that might be applying here.

Reply:
Thanks, that looks like a good style guide. I don't think we're using anything as perscriptive in PF.
PTAL and help me map to it.

Reply:
Yep, I think this is already conformant from what I can see - the only discrepancy is the hierarchy per other comment. Would typically expect to see the pattern of `/noun1s/{noun1_id}/noun2s/{noun2_id}` etc. FOr now I might just flatten this to `/entities:generateSqlQuery`, if that seems like the most relevant noun